tests/framework-tests/preferences/js/CompositePanel.js

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
Shall this file be named "CompositePanelTests.js" or "CompositePanelTestsGrades.js" to be in sync with other test files?